frappe/public/js/frappe/widgets/shortcut_widget.js

```diff
@@ -49,12 +49,7 @@ export default class ShortcutWidget extends Widget {
 			}
 
 			if (this.type == "URL") {
-				if (frappe.open_in_new_tab) {
-					window.open(this.url, "_blank");
-					frappe.open_in_new_tab = false;
-				} else {
-					window.location.href = this.url;
-				}
+				window.open(this.url, "_blank");
```

Comment:
I don't think it's specified that this is always an external URL. I'd suggest if URL is external here. 

`ctrl+click` does open things in external window btw.

Reply:
true - but I'm guessing the main intention was to open external links (judging from the [pr](https://github.com/frappe/frappe/pull/20769) description)? (even if that's not all it does) - made sense to me to open in a new tab by default in that case